Running-Mastodon/Production-guide.md

```diff
@@ -37,7 +37,9 @@ server {
   server_name example.com;
 
   ssl_protocols TLSv1.2;
-  ssl_ciphers HIGH:!MEDIUM:!LOW:!aNULL:!NULL:!SHA;
+  ssl_ciphers 'ECDHE+CHACHA20:ECDHE+AESGCM'; # or 'ECDHE:!DSS:!PSK:!SRP:!3DES:!RC4:!DES:!IDEA:!RC2:!NULL'
```

Comment:
`EECDH+AESGCM` is pretty useless in this case. `EECDH+AES` already order correctly the cipher suite (AEAD first then non-AEAD, AES256 first then AES128, with SHA1 at the end.
The only difference is AESGCM128, which is a little prefered on the `EECDH+AES` form (before AES256CCM) than on the `EECDH+AESGCM:EECDH+AES` (after AES256CCM).
I personally prefer the short form (`EECDH+AES`), which order AES256 in any form over AES128.